Library/ViewModels/PledgeViewModel.swift

```diff
@@ -94,8 +127,12 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
     }
     .skipNil()
 
-    self.continueViewHidden = isLoggedIn
-    self.paymentMethodsViewHidden = isLoggedIn.negate()
+    self.continueViewHidden = Signal
+      .combineLatest(isLoggedIn, context)
+      .map { $0 || $1.isUpdate }
+
+    self.paymentMethodsViewHidden = Signal.combineLatest(isLoggedIn, context)
+      .map { !$0 || $1.isUpdate }
 
     let paymentSourceSelected = Signal.combineLatest(
```

Comment:
What about all the payment source stuff? I think we would want to prevent the payment methods section from being configured, because if it is then it would make an unnecessary request for the user's cards. Same with the Stripe SDK configuration stuff, although that's more innocuous.

Reply:
Oh yeah for sure - will also filter those and add tests for these things today.